src/components/resource/components/edit/resource-settings/edit-resource-settings.js

```diff
@@ -65,14 +67,30 @@ const EditResourceSettings = ({
     >
       {renderFields
         ? (
-          <>
-            <VisibilityField disabled={visibilityMessage} />
-            <div>
-              {hasInheritedProxy && getAccordionContent()}
-              {model.package.isCustom && <CustomUrlFields />}
-              <AccessTypeEditSection accessStatusTypes={accessStatusTypes} />
-            </div>
-          </>
+          <Row>
+            <Col xs={3}>
+              <VisibilityField disabled={visibilityMessage} />
+            </Col>
+            <Col xs={9}>
+              <div>
```

Comment:
Is it necessary `<div>`?

Reply:
You're right, it was redundant, removed now